crates/db_schema/src/schema.rs

```diff
@@ -444,6 +444,7 @@ diesel::table! {
         post_listing_mode -> PostListingModeEnum,
         totp_2fa_enabled -> Bool,
         enable_keyboard_navigation -> Bool,
+        enable_animated_avatars -> Bool,
     }
```

Comment:
I'd just add this line manually, and nothing else. The formatting changes in this file could potentially break some of the git patches done to it.